test_conformance/api/test_api_min_max.cpp

```diff
@@ -1667,8 +1668,8 @@ int test_min_max_constant_buffer_size(cl_device_id deviceID, cl_context context,
         /* Test running the kernel and verifying it */
         threads[0] = numberOfInts;
         localThreads[0] = 1;
-        log_info("Filling constant buffer with %d cl_ints (%d bytes).\n",
-                 (int)threads[0], (int)(threads[0] * sizeof(cl_int)));
+        log_info("Filling constant buffer with %lld cl_ints (%lld bytes).\n",
+                 threads[0], threads[0] * sizeof(cl_int));
```

Comment:
```suggestion
        log_info("Filling constant buffer with %zu cl_ints (%zu bytes).\n",
                 threads[0], threads[0] * sizeof(cl_int));
```

Pedantic: `threads` is a `size_t`.